src/QuantumToolbox.jl

```diff
@@ -35,8 +35,11 @@ import DiffEqCallbacks: PeriodicCallback, PresetTimeCallback, TerminateSteadySta
 import OrdinaryDiffEqCore: OrdinaryDiffEqAlgorithm
 import OrdinaryDiffEqTsit5: Tsit5
 
+using StochasticDiffEq
```

Comment:
maybe use `import` instead of `using`?
seems that we only need:
```julia
import SciMLBase: SDEProblem # add this one above
import StochasticDiffEq: StochasticDiffEqAlgorithm, SRA1
```